enos/enos-scenario-e2e-docker-base-with-vault.hcl

```diff
@@ -75,7 +80,8 @@ scenario "e2e_docker_base_with_vault" {
     ]
     variables {
       image_name       = matrix.builder == "crt" ? var.boundary_docker_image_name : step.build_boundary_docker_image.image_name
-      network_name     = step.create_docker_network.network_name
+      network_name     = [local.network_cluster]
+      database_network = local.network_cluster
```

Comment:
Added this variable since the boundary (controller) container could be added to multiple networks, but we need to know which one is the database one so that we can initialize the database. In this scenario, they happen to be on the same network, but in other scenarios, we may isolate it so that the database is in a separate network from the rest of the cluster.